init/stripe.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright 2023 the Deno authors. All rights reserved. MIT license.
+import type { Stripe } from "stripe";
+import { SITE_DESCRIPTION } from "@/constants.ts";
+import "std/dotenv/load.ts";
+import { stripe } from "@/utils/stripe.ts";
+
+async function createPremiumTierProduct(stripe: Stripe) {
+  return await stripe.products.create({
+    name: "Premium tier",
```

Comment:
I'm wondering if the stuff in this object should be configurable? Does that make any sense?

Reply:
Good suggestion. Though, we'll leave this as-is for now in the spirit of prioritising ease of setup over customizability.